client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -179,15 +205,8 @@ export default React.createClass( {
 		}
 	},
 	render: function() {
-		let pipe = this.getPipe();
-
-		for ( let renderer of pipe ) {
-			let result = renderer();
-			if ( result ) {
-				return result;
-			}
-		}
-		return null;
+		const notices = this.getPipe().map( renderer => renderer() ).filter( notice => notice );
```

Comment:
I'm proposing this for real. If 1 domain is expiring tomorrow, another expired a week ago, and 2 more are unverified and will be suspended in 5 days, all four of the messages are critical.